src/Tests/dotnet-watch.Tests/GlobbingAppTests.cs

```diff
@@ -128,9 +129,9 @@ public async Task ChangeExcludedFile()
             var changedFile = Path.Combine(app.SourceDirectory, "exclude", "Baz.cs");
             File.WriteAllText(changedFile, "");
 
-            var restart = app.HasRestarted();
-            var finished = await Task.WhenAny(Task.Delay(TimeSpan.FromSeconds(5)), restart);
-            Assert.NotSame(restart, finished);
+            var fileChanged = app.HasFileChanged();
+            var finished = await Task.WhenAny(Task.Delay(TimeSpan.FromSeconds(5)), fileChanged);
+            Assert.NotSame(fileChanged, finished);
```

Comment:
This was catching the [initial start message](https://github.com/dotnet/sdk/blob/main/src/Assets/TestProjects/WatchGlobbingApp/Program.cs#L16) from the sample app, checking if a hot reload file change event happened is probably more representative anyway.

Was getting:
```
 1/7/2022 9:46:13 PM: starting process: '/workspaces/sdk/artifacts/bin/redist/Debug/dotnet/dotnet watch run --'
 1/7/2022 9:46:13 PM: process started: '/workspaces/sdk/artifacts/bin/redist/Debug/dotnet/dotnet watch run --'
 Waiting for output line [msg == 'dotnet watch 🚀 Started']. Will wait for 120 sec.
 1/7/2022 9:46:14 PM: post: 'dotnet watch ⌚ Polling file watcher is enabled'
 1/7/2022 9:46:14 PM: recv: 'dotnet watch ⌚ Polling file watcher is enabled'. Does not match condition '[msg == 'dotnet watch 🚀 Started']'.
 1/7/2022 9:46:16 PM: post: 'dotnet watch 🚀 Started'
 1/7/2022 9:46:16 PM: recv: 'dotnet watch 🚀 Started'. Matches condition '[msg == 'dotnet watch 🚀 Started']'.
 Waiting for output line [msg == 'Started']. Will wait for 30 sec.
 1/7/2022 9:46:18 PM: post: 'Started'
 1/7/2022 9:46:18 PM: recv: 'Started'. Matches condition '[msg == 'Started']'.
 Disposing WatchableApp
```